### ISSUE TITLE
flaky test: bad test check in box/alter_limits.test.lua test

### ISSUE THREAD

**Issue by avtikhon** [tarantool/tarantool-qa#115]
OS: all
OS Version: all
Tarantool version: master (only!)

Found in box/alter_limits.test.lua test that it has hard coded results which in real can be different, check different runs:
1. https://github.com/tarantool/tarantool/runs/2339969973#step:5:13039
```
[018] @@ -739,8 +739,8 @@
[018]  s.index.third:rename('second')
[018]  ---
[018]  - error: 'Duplicate key exists in unique index "name" in space "_index" with old tuple
[018] -    - [814, 1, "second", "tree", {"unique": true}, [[1, "string"]]] and new tuple
[018] -    - [814, 2, "second", "hash", {"unique": true}, [[2, "unsigned"]]]'
[018] +    - [532, 1, "second", "tree", {"unique": true}, [[1, "string"]]] and new tuple
[018] +    - [532, 2, "second", "hash", {"unique": true}, [[2, "unsigned"]]]'
```
2. https://github.com/tarantool/tarantool/runs/2339969886#step:5:12080
```
[003] @@ -739,8 +739,8 @@
[003]  s.index.third:rename('second')
[003]  ---
[003]  - error: 'Duplicate key exists in unique index "name" in space "_index" with old tuple
[003] -    - [814, 1, "second", "tree", {"unique": true}, [[1, "string"]]] and new tuple
[003] -    - [814, 2, "second", "hash", {"unique": true}, [[2, "unsigned"]]]'
[003] +    - [597, 1, "second", "tree", {"unique": true}, [[1, "string"]]] and new tuple
[003] +    - [597, 2, "second", "hash", {"unique": true}, [[2, "unsigned"]]]'
```
3. https://github.com/tarantool/tarantool/runs/2339969740#step:5:12576
```
[029] @@ -739,8 +739,8 @@
[029]  s.index.third:rename('second')
[029]  ---
[029]  - error: 'Duplicate key exists in unique index "name" in space "_index" with old tuple
[029] -    - [814, 1, "second", "tree", {"unique": true}, [[1, "string"]]] and new tuple
[029] -    - [814, 2, "second", "hash", {"unique": true}, [[2, "unsigned"]]]'
[029] +    - [547, 1, "second", "tree", {"unique": true}, [[1, "string"]]] and new tuple
[029] +    - [547, 2, "second", "hash", {"unique": true}, [[2, "unsigned"]]]'
```

Also found that this issue became available after the [commit](https://github.com/tarantool/tarantool/commit/d11fb3061e15faf4e0eb5375fb8056b4e64348ae) where hard coded numbers were added to results files.

Test passed in CI after [commit](https://github.com/tarantool/tarantool/commit/d11fb3061e15faf4e0eb5375fb8056b4e64348ae) because running in groups after 'box/alter.test.lua' test ID space number became equal to '814' as was needed, try:
`./test-run.py box/alter.test.lua box/alter_limits.test.lua`
But running test standalone found that ID space number was not enough and is equal to '525', try:
`./test-run.py box/alter_limits.test.lua`
It happened because spaces increases the ID space number on creation and does not decrease it on drop.

It means that the check in test need to be made universal blocking the ID space number from checking with results filter.

**Comment by Totktonada** [comment]
Fixed by https://github.com/tarantool/tarantool/pull/6003?

Please, re-check and close, if it is so.

**Comment by avtikhon** [comment]
> Fixed by [tarantool/tarantool#6003](https://github.com/tarantool/tarantool/pull/6003)?
> 
> Please, re-check and close, if it is so.

Right, I've checked the fix within PR #6003, it helped, closing.
